com.hunterwittenborn.Celeste.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.hunterwittenborn.Celeste
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+command: celeste
+finish-args:
+  # We need to display on Wayland/X11.
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  # Enable OpenGL rendering.
+  - '--device=dri'
+  # Users may want to sync files from anywhere on their system.
+  - '--filesystem=host'
+  # We need network access in order to sync files to remotes.
+  - '--share=network'
+  # Used for X11 performance as described at https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f1.
+  - '--share=ipc'
+  # I haven't been able to find out why, but we have to have access to the full
+  # DBus session or the tray stops running due to not being able to send
+  # messages to the main application (the tray app was designed to quit when it
+  # can't any messages to the main application). Neither the Debian package nor
+  # the Snap are affected by the issue, so I'm not quite sure what's up with the
+  # Flatpak.
+  - '--socket=session-bus'
```

Comment:
i'll be clear, blanket permssion to the session-bus is NOT allowed.

Reply:
I'm not really wanting it either, I just can't get it to work without it. I know it's a lot less secure, I just can't find what might be causing issues without it. I haven't found any DBus names that aren't specified when using `flatpak run --log-session-bus`, so I'm kind of just at a dead end right now honestly.

Reply:
I'm going to keep looking into trying to figure out what's wrong, I just don't know what else I might want to do